app/src/test/java/com/kickstarter/viewmodels/CreatorDashboardViewModelTest.java

```diff
@@ -154,6 +156,8 @@ public void testProjectsForBottomSheet_WithManyProjects() {
 
   @Test
   public void testProjectSwitcherProjectClickOutput() {
+    DateTimeUtils.setCurrentMillisFixed(new DateTime().getMillis());
```

Comment:
👀  This is needed for the test to keep working, it was done before on KSRobolectricTestCase.java:51 it has been isolated just for the concrete test.